sound/soc/sof/intel/bdw.c

```diff
@@ -709,6 +709,37 @@ static int bdw_remove(struct snd_sof_dev *sdev)
 	return 0;
 }
 
+static const struct snd_soc_dai_ops bdw_dai_ops = {
+};
+
+#define BDW_FORMATS (SNDRV_PCM_FMTBIT_S16_LE | SNDRV_PCM_FMTBIT_S24_LE | \
+	SNDRV_PCM_FMTBIT_S32_LE | SNDRV_PCM_FMTBIT_FLOAT)
```

Comment:
do we support FLOAT on BDW? (same comments for all machines btw, I don't think we can deal with FLOAT on a DAI.

Reply:
This is also initiated by Liam, Liam, do you have comment on it?